exchanges/exchange.go

```diff
@@ -262,7 +262,7 @@ type IBotExchange interface {
 
 	GetFundingHistory() ([]FundHistory, error)
 	SubmitOrder(p pair.CurrencyPair, side OrderSide, orderType OrderType, amount, price float64, clientID string) (SubmitOrderResponse, error)
-	ModifyOrder(orderID int64, modify ModifyOrder) (int64, error)
+	ModifyOrder(action ModifyOrder) ModifyOrderResponse
```

Comment:
Since this is for a single order and to be consistent with other wrapper signatures, could you change the response signature to be (ModifyOrderResponse, error)?

Reply:
Yeah will do

Reply:
👍 looks good to merge after this change is done. Great work @shazbert